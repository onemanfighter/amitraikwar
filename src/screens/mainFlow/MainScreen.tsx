const MainScreen = () => {
  return (
    <div>
      <h1>Main Screen</h1>
    </div>
  );
};

export default MainScreen;
