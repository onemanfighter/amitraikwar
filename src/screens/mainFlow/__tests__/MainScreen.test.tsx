import { render } from '@testing-library/react';
import MainScreen from '../MainScreen';

describe('MainScreen', () => {
  it('should render correctly', () => {
    const { container } = render(<MainScreen />);

    expect(container).toMatchSnapshot();
  });
});
