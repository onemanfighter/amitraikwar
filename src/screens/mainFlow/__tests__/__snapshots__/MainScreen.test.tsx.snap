// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainScreen should render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <span
      class="index-module_type__E-SaG"
      style="font-size: 2em; display: inline-block;"
    />
    <p
      class="chakra-text css-0"
    >
      MainScreen
    </p>
  </div>
</div>
`;
